common/static/coffee/src/discussion/utils.coffee

```diff
@@ -113,7 +113,7 @@ class @DiscussionUtil
       alertDiv = $("<div class='modal' role='alertdialog' id='discussion-alert' aria-describedby='discussion-alert-message'/>").css("display", "none")
       alertDiv.html(
         "<div class='inner-wrapper discussion-alert-wrapper'>" +
-        "  <button class='close-modal dismiss' title='Close modal'><span class='icon fa fa-times' aria-hidden='true'></span></button>" +
+        "  <button class='close-modal dismiss' title='" + gettext("Close") + "'><span class='icon fa fa-times' aria-hidden='true'></span></button>" +
```

Comment:
If you look at the block of code, you'll see that concatenation like this was already happening, later in the same string. Given that, I don't feel like this needs to be fixed as part of this PR.